Chargebee/Classes/Purchase/CBPurchaseManager.swift

```diff
@@ -105,14 +105,14 @@ public extension CBPurchase {
     @available(*, deprecated, message: "This will be removed in upcoming versions, Please use this API func purchaseProduct(product: CBProduct, customer : CBCustomer? = nil, completion)")
     func purchaseProduct(product: CBProduct, customerId : String? = "" ,completion handler: @escaping ((_ result: Result<(status:Bool, subscriptionId:String?, planId:String?), Error>) -> Void)) {
         buyProductHandler = handler
-        activeProduct = product.product
+        activeProduct = product
         self.customer = CBCustomer(customerID: customerId ?? "")
```

Comment:
Should this be `""` or null?

Reply:
if `customerId = ""` then  `self.customer` object will be nill internally `customerId` will be `""` only in `CBCustomer` `customerId` we configured as optional with default empty `""` value